ember-primitives/src/helpers/link.ts

```diff
@@ -80,7 +90,7 @@ function isActive(router: RouterService, href: string, includeQueryParams?: bool
 
   if (!currentPath) return false;
 
-  if (hrefPath !== currentPath) return false;
+  if (activeOnSubPaths ? !currentPath.startsWith(hrefPath) : hrefPath !== currentPath) return false;
```

Comment:
```suggestion
  if (activeOnSubPaths && !currentPath.startsWith(hrefPath)) return false;
  if (hrefPath !== currentPath) return false;
```
is this equiv?

I think it reads a bit better than the in-condition ternary?

Reply:
or no, I suppose it wouldn't be. hmm. nevermind